eisbuk-admin/functions/package.json

```diff
@@ -31,8 +31,8 @@
     "@firebase/testing": "^0.20.11",
     "@types/luxon": "^1.27.0",
     "@types/uuid": "^8.3.0",
-    "@typescript-eslint/eslint-plugin": "^4.28.4",
-    "@typescript-eslint/parser": "^4.28.4",
+    "@typescript-eslint/eslint-plugin": "^4.1.1",
+    "@typescript-eslint/parser": "^4.1.1",
```

Comment:
Why downgrade this? Or did it come automatically with firebase updates?

Reply:
I don't know why it got downgraded. I kindly asked `yarn` to do the upgrades for me, but didn't actually check what it did. Now I manually edited `package.json` to set the version to `4.30.0` and ran another round of `yarn upgrade`.